packages/react-core/src/components/Slider/Slider.tsx

```diff
@@ -61,6 +61,10 @@ export interface SliderProps extends Omit<React.HTMLProps<HTMLDivElement>, 'onCh
   leftActions?: React.ReactNode;
   /** Actions placed to the right of the slider */
   rightActions?: React.ReactNode;
+  /** Id to use for Slider descriptor */
+  'aria-describedby'?: string;
+  /** Id to use for Slider label */
+  'aria-labelledby'?: string;
```

Comment:
The descriptions could be tweaked here to reflect that multiple IDs could be passed in, as well as what they're being attached to:

```suggestion
  /** One or more id's to use for the slider thumb description */
  'aria-describedby'?: string;
  /** One or more id's to use for the slider thumb label */
  'aria-labelledby'?: string;
```